signer/src/testing/stacks.rs

```diff
@@ -26,6 +28,21 @@ pub const DUMMY_SORTITION_INFO: SortitionInfo = SortitionInfo {
     committed_block_hash: None,
 };
 
+/// Some dummy tenure info
+pub const DUMMY_TENURE_INFO: RPCGetTenureInfo = RPCGetTenureInfo {
+    consensus_hash: ConsensusHash([0; 20]),
+    tenure_start_block_id: StacksBlockId([0; 32]),
+    parent_consensus_hash: ConsensusHash([0; 20]),
+    parent_tenure_start_block_id: StacksBlockId([
+        0x55, 0xc9, 0x86, 0x1b, 0xe5, 0xcf, 0xf9, 0x84, 0xa2, 0x0c, 0xe6, 0xd9, 0x9d, 0x4a, 0xa6,
+        0x59, 0x41, 0x41, 0x28, 0x89, 0xbd, 0xc6, 0x65, 0x09, 0x41, 0x36, 0x42, 0x9b, 0x84, 0xf8,
+        0xc2, 0xee,
+    ]), // StacksBlockId::first_mined()
```

Comment:
Curious, why the change? And what is the significance of this block?

Also, we should remove the comment right?

Reply:
We cannot use `StacksBlockId::first_mined()` since is not a const fn; those bytes are the result of calling `StacksBlockId::first_mined()`. I'll nit the comment to avoid confusion about it.

Reply:
Done in [e6ff96b](https://github.com/stacks-network/sbtc/pull/1058/commits/e6ff96b95dc0bc93ff2d531d62eb859e3df1d60c)